### ISSUE TITLE
Big thank you for the hover-preview feature

### ISSUE THREAD

**Issue by lexruee** [franglais125/dash-to-dock#11]
@franglais125 
I've installed and tested your hover_preview branch (https://github.com/franglais125/dash-to-dock/commit/b850650136c419535fba010129969623400b80c0), and it works flawless on my Debian Sid/Gnome3.22 system. 

Just want to say thank you for this awesome feature. I hope this will be merged soon into the master branch and next release of dash-to-dock. 

Keep up your good work for dash-to-dock!

![hover-preview](https://user-images.githubusercontent.com/649340/28315587-574563dc-6bbf-11e7-895c-79fc823a8812.png)

**Comment by franglais125** [comment]
Awesome! Happy you like it :). You might want to update the branch every couple of days, as I might be pushing changes/fixes.

If you find issues, please let me know so I can fix them!
Cheers

**Comment by lexruee** [comment]
@franglais125 
Currently, I found only one issue when I use intellihide and autohide together with the hover preview. But I'm not sure if this is an intended behaviour.

When I move the mouse pointer to the window preview, then the dock is fading out. 
As a user I would expect the dock and the window preview to be visible until I move the mouse pointer somewhere else.

I made a short youtube video that demonstrates this effect: https://youtu.be/jnUMQlCbuCI

**Comment by franglais125** [comment]
Yup, absolutely, something to be fixed. I thought it was taken care of. Thanks!

**Comment by franglais125** [comment]
It took me longer than expected (other small issues in the way), but I finally fixed the intellihide problem. Thanks for reporting that!

Let me know how it goes. Merci d'avance :)

**Comment by lexruee** [comment]
@franglais125 
Nice :-) Now  it works perfectly. 
I see the fix was to introduce a boolean flag dontClose :smile:: [docking.js#L690](https://github.com/franglais125/dash-to-dock/blob/c21aaf2f790203cd36812312060e57c2338ac583/docking.js#L690)

Merci beaucoup :smiley:
